docs/guides/integration-strategy.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Strategy on how to integrate Bearer into your workflow
+---
+
+# Strategy on how to integrate Bearer into your workflow
+
+Welcome to this guide on integrating Bearer into your development process. 
+
+Follow these steps to seamlessly incorporate Bearer at every stage of your workflow. We understand that adopting new practices can take time, so we provide this guide as a strategy blueprint to gradually implement these changes, allowing you and your team to adjust easily.
+
+## Step 1: Understand the current state of your application
+
+Before integrating Bearer into your CI/CD pipeline, it's essential to assess the existing landscape. To do so, follow these steps:
+- Install Bearer on your local machine (see [doc](/quickstart)).
+- Run the scan command on your repository main branch.
+
+This initial scan will provide insights into your application's current state. Let's dive in!
+
+```bash
+$ bearer scan myRepo/
+…
+MEDIUM: Weak hashing library (MD5) detected. [CWE-327, CWE-328]
+https://docs.bearer.com/reference/rules/javascript_lang_weak_hash_md5
+To ignore this finding, run: bearer ignore add 42a76a8c10a52b38c1b8729a2f211830_0
+
+File: lib/insecurity.ts:43
+
+ 43 export const hash = (data: string) => crypto.createHash('md5').update(data).digest('hex')
+=====================================
+
+72 checks, 55 findings
+
+CRITICAL: 0
+HIGH: 21 (CWE-22, CWE-312, CWE-79, CWE-798, CWE-89, CWE-918)
+MEDIUM: 34 (CWE-327, CWE-328, CWE-525, CWE-548, CWE-73, CWE-79)
+LOW: 0
+WARNING: 0
+```
+
+If your code triggers any findings, as show above, read the next section. Otherwise, skip directly to step 3.
+
+## Step 2: Triage and remediate existing findings
+
+We need to start triaging findings from the CLI on your local machine. 
+
+Here is our recommended strategy:
+1. Start by addressing "Critical" findings, then "High", "Medium", and finally "Low" findings.
+2. Remove any irrelevant findings for each severity level by using the ```bearer ignore``` command and categorize them as *false positive* (see [doc](/guides/configure-scan/#ignore-specific-findings)).
+3. If a rule is problematic with your code, skip it entirely using the ```--skip-rule``` option (see [doc](/guides/configure-scan/#skip-rules-for-the-entire-scan)).
```

Comment:
Small suggestion - reads a bit clearer for me but you can take it or leave it. 
```suggestion
3. If a rule is problematic for your codebase, skip it entirely using the ```--skip-rule``` option (see [doc](/guides/configure-scan/#skip-rules-for-the-entire-scan)).
```